server.js

```diff
@@ -3202,6 +3202,37 @@ cache(function (data, match, sendBadge, request) {
   });
 }));
 
+// Docker Hub stars integration.
+camp.route(/^\/docker\/stars\/([^\/]+)\/([^\/]+)\.(svg|png|gif|jpg|json)$/,
+cache(function(data, match, sendBadge, request) {
+  var user = match[1];  // eg, _ or mashape
+  var repo = match[2];  // eg, mockbin
+  var format = match[3];
+  // Convert underscore to library for api path
+  user = user === '_' ? 'library' : user;
+  var baseURL = 'https://registry.hub.docker.com/v2/';
+  var path = 'repositories/'+user+'/'+repo+'/stars/count/';
+  var badgeData = getBadgeData('docker', data);
+  request(baseURL+path, function(err, res, buffer) {
+    if (err != null) {
+      badgeData.text[1] = 'inaccessible';
+      sendBadge(format, badgeData);
```

Comment:
Where exactly? Some integrations (For example the DUB license and version integration above) don't have any return statements.